src/api/app/views/webui/project/pulse.html.haml

```diff
@@ -0,0 +1,20 @@
+- @pagetitle = "Pulse of #{@project}"
+- project_bread_crumb 'Pulse'
+
+
+= render partial: 'webui/project/tabs'
+%h3 Pulse of #{@project}
+- @pulse.each do |log_entry|
+  %p
+    &bull;
+    = log_entry.event_type.camelcase.prepend('Event::').constantize.description
+    - if log_entry.package_name
+      (#{link_to(log_entry.package_name, package_show_path(@project.name, log_entry.package_name))})
+    - if log_entry.user_name
+      by #{link_to(log_entry.user_name, user_show_path(log_entry.user_name))}
+    - if log_entry.bs_request_id
+      - number = BsRequest.find(log_entry.bs_request_id).number
```

Comment:
This is not needed I guess,  `ProjectLogEntry` model have a relationship with `bs_request`. You can just do `log_entry.bs_request.number`, right?